src/gui_common/thrive_theme.tres

```diff
@@ -165,7 +165,7 @@ thickness = 5
 content_margin_left = 13.0
 content_margin_top = 2.0
 
-[sub_resource type="StyleBoxFlat" id=21]
+[sub_resource type="StyleBoxFlat" id=41]
```

Comment:
These seem like incorrect ID changes. Are there any intended changes in this file?

Reply:
I've did git restore on this file but godot insisted on changing these IDs.

Reply:
Doesn't do it for me. Testing on this branch, Godot automatically fixes the IDs for me:

![Kuvakaappaus - 2021-11-21 20-23-35](https://user-images.githubusercontent.com/3796411/142774298-1857a762-4717-43d8-b999-296a04d16223.png)

Reply:
Actually, I'll push that change to this branch, let me know if that causes problems.

Reply:
Looks good.

Reply:
Scratch that, looks like this breaks the styling for species name's LineEdit. The thrive theme got automatically changed last time, so maybe my ID changes is the right one?